openfisca_core/columns.py

```diff
@@ -32,6 +37,7 @@ class Column(object):
     formula_class = None
     is_period_size_independent = False  # When True, value of column doesn't depend from size of period (example: age)
     is_permanent = False  # When True, value of column doesn't depend from time (example: ID, birth)
+    period_behavior = None
```

Comment:
I'm still not comfortable with the term `period_behavior`.

What about keeping the same name `period_unit` like it is used in `periods.py`? I know this is less "searchable" but this *is* the same concept.

Moreover the `periods.py` should use the same `Enum` (cf my previous comment).